lib/zinnia.js

```diff
@@ -121,36 +126,45 @@ export async function run ({
     }
   }
 
-  const childProcess = execa(zinniadExe, modules, {
-    cwd: moduleBinaries,
-    env: {
-      FIL_WALLET_ADDRESS,
-      STATE_ROOT,
-      CACHE_ROOT
-    }
-  })
+  const childProcesses = []
+  let exitReason
```

Comment:
We need to track the exit reason for each process.

Reply:
Fixed in https://github.com/filecoin-station/core/pull/323/commits/22b89c79a1e7d8f802dce7db385c59bfa748e0d8, `exitReason`'s scope was wrong